remo/profiles/models.py

```diff
@@ -251,9 +252,9 @@ def __unicode__(self):
 
     def save(self, *args, **kwargs):
         # Save the timestamp when a Rep becomes available
-        if not self.id:
+        if not self.id and self.start_date <= get_date():
             self.is_unavailable = True
-        else:
+        elif self.start_date < get_date():
```

Comment:
Will this prevent a user for getting back available if she chooses so on the same day as `start_date`?